plugin.js

```diff
@@ -682,189 +496,423 @@
 		};
 	}
 
-	placeholdersCollection = new PlaceholdersCollection();
+	/**
+	 * @class
+	 * @constructor
+	 * @param {CKEDITOR.editor} editor
+	 */
+	function PlaceholderElementsPlugin(editor)
+	{
+		var _this = this;
```

Comment:
Why? Especially why using it for property declarations? Did you encounter any weird scoping issues?